container/src/services/webcomponents.service.ts

```diff
@@ -39,24 +41,65 @@ export class WebComponentService {
     }
   }
 
+  /**
+   * Function that uses the current instance of the containerService to dispatch a Luigi event to the current instance of the container
+   * that is 'thisComponent'
+   * @param msg the message to be delivered
+   * @param data the data to be sent
+   * @param callback the callback function to be called
+   */
+  dispatchLuigiEvent(msg: string, data: any, callback?: Function) {
+    this.containerService.dispatch(msg, this.thisComponent, data, callback);
+  }
+
+  /**
+   * This function is used to create the LuigiClient API for the webcomponent based microfrontend.
+   * As the function expands with more functionality it might be moved to a separate class.
+   * @param eventBusElement the event bus to be used for cross webcomponent communication, i.e.: for compound microfrontends container scenario
+   * @param nodeId refers to an attribute of the webcomponent to identify from the rest
+   * @param wc_id a tagname that is used when creating the webcomponent element
+   * @returns an object with the LuigiClient API

```

Comment:
```suggestion
   * This function is used to create the Luigi Client API for the webcomponent-based microfrontend.
   * As the function expands with more functionality, it might be moved to a separate class.
   * @param eventBusElement the event bus to be used for cross webcomponent communication, i.e.: for compound microfrontends container scenario
   * @param nodeId refers to an attribute of the webcomponent to be identified from the rest
   * @param wc_id a tagname that is used when creating the webcomponent element
   * @returns an object with the Luigi Client API

```

Reply:
I commited this, but what is the change suggested, is it the blank line in the end ? @alexandra-simeonova